js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -560,7 +560,8 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
             recursiveExpansion: this.option('expandNodesRecursive'),
             selectionRequired: this.option('selectionRequired'),
             dataType: this.option('dataStructure'),
-            sort: this._dataSource && this._dataSource.sort()
+            sort: this._dataSource && this._dataSource.sort(),
```

Comment:
Minor comment

Can we use the ```?``` here?
```suggestion
            sort: this._dataSource?.sort(),
```